independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json

```diff
@@ -0,0 +1,311 @@
+        },
+        "summary": "Synthesize",
+        "parameters": [
+          {
+            "name": "voice",
+            "in": "query",
+            "x-ms-summary": "Voice Name",
+            "required": false,
+            "type": "string",
+            "default": "en-US_MichaelV3Voice",
+            "description": "Voice Name",
+            "enum": [
+              "ar-MS_OmarVoice",
+              "cs-CZ_AlenaVoice",
+              "de-DE_BirgitV3Voice",
+              "de-DE_DieterV3Voice",
+              "de-DE_ErikaV3Voice",
+              "en-AU_CraigVoice",
+              "en-AU_MadisonVoice",
+              "en-AU_SteveVoice",
+              "en-GB_CharlotteV3Voice",
+              "en-GB_JamesV3Voice",
+              "en-GB_KateV3Voice",
+              "en-US_AllisonExpressive",
+              "en-US_AllisonV3Voice",
+              "en-US_EmilyV3Voice",
+              "en-US_EmmaExpressive",
+              "en-US_HenryV3Voice",
+              "en-US_KevinV3Voice",
+              "en-US_LisaExpressive",
+              "en-US_LisaV3Voice",
+              "en-US_MichaelExpressive",
+              "en-US_MichaelV3Voice",
+              "en-US_OliviaV3Voice",
+              "es-ES_EnriqueV3Voice",
+              "es-ES_LauraV3Voice",
+              "es-LA_SofiaV3Voice",
+              "es-US_SofiaV3Voice",
+              "fr-CA_LouiseV3Voice",
+              "fr-FR_NicolasV3Voice",
+              "fr-FR_ReneeV3Voice",
+              "it-IT_FrancescaV3Voice",
+              "ja-JP_EmiV3Voice",
+              "ko-KR_HyunjunVoice",
+              "ko-KR_SiWooVoice",
+              "ko-KR_YoungmiVoice",
+              "ko-KR_YunaVoice",
+              "nl-BE_AdeleVoice",
+              "nl-BE_BramVoice",
+              "nl-NL_EmmaVoice",
+              "nl-NL_LiamVoice",
+              "pt-BR_IsabelaV3Voice",
+              "sv-SE_IngridVoice",
+              "zh-CN_LiNaVoice",
+              "zh-CN_WangWeiVoice",
+              "zh-CN_ZhangJingVoice"
+            ]
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "x-ms-summary": "Text",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "text": {
+                  "type": "string",
+                  "description": "Text to convert to speech",
+                  "title": "Text"
+                }
+              },
+              "required": [
+                "text"
+              ]
+            }
+          }
+        ],
+        "description": "Synthesizes text to audio that is spoken in the specified voice. The service bases its understanding of the language for the input text on the specified voice. Use a voice that matches the language of the input text.",
+        "operationId": "Synthesize"
+      }
+    },
+    "/v1/voices": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```

Comment:
@lucastitus empty schema is not allowed, please remove it or define a proper schema.